plugins/VstBase/RemoteVstPlugin.cpp

```diff
@@ -842,7 +842,6 @@ void RemoteVstPlugin::initEditor()
 #endif
 
 #else
-	XEvent e;
```

Comment:
Since `e` 's declaration is removed, won't this cause any issues below?
But contradictory to my assumption, the build succeeds.

Reply:
`e` was unused, so I had to remove it to fix the `unused-variable` warning.

Reply:
Ohh i see. I think i saw something related to e in a line below.